src/sdk/config_fs.js

```diff
@@ -151,88 +218,184 @@ class ConfigFS {
-     * get_root_account_path_by_name returns the full account path by name
+     * get_account_path_by_name returns the full account path by name
+     * root account can be found by name under accounts_by_name/account_name.symlink
      * @param {string} account_name
      * @returns {string} 
      */
-    get_root_account_path_by_name(account_name) {
-        // TODO -  change to this.symlink(account_name) on identities/ PR;
-        return path.join(this.accounts_dir_path, this.json(account_name));
+    get_account_path_by_name(account_name) {
+        return path.join(this.accounts_by_name_dir_path, this.symlink(account_name));
     }
 
     /**
-     * get_iam_account_path_by_name returns the full account path by name
+     * get_user_path_by_name returns the full iam user path by name
+     * iam user can be found by name under identities/owner_account_id/users/iam_account_name.symlink
      * @param {string} account_name
-     * @param {string} owner_root_account_id
+     * @param {string} owner_account_id
      * @returns {string} 
     */
-    get_iam_account_path_by_name(account_name, owner_root_account_id) {
-       // TODO -  change to this.symlink(account_name) on identities/ PR;
-       // update IAM user location identities/root_id_number/iam_account_name.symlink
-       return path.join(this.accounts_dir_path, this.json(account_name));
+    get_user_path_by_name(account_name, owner_account_id) {
+        // TODO - change to return path.join(this.identities_dir_path, owner_account_id, 'users', this.symlink(account_name));
+        return path.join(this.accounts_by_name_dir_path, this.symlink(account_name));
     }
 
     /**
-     * get_account_relative_path_by_name returns the full account path by name
+     * get_old_account_relative_path_by_name returns the old (5.17) full account path by name
      * @param {string} account_name
      * @returns {string} 
     */
-    get_account_relative_path_by_name(account_name) {
-       // TODO -  change to this.symlink(account_name) on identities/ PR;
-       return path.join('../', CONFIG_SUBDIRS.ACCOUNTS, this.json(account_name));
+    get_old_account_relative_path_by_name(account_name) {
+        return path.join('../', CONFIG_SUBDIRS.ACCOUNTS, this.json(account_name));
     }
 
     /**
-     * get_account_path_by_access_key returns the full account path by access key
+     * get_account_relative_path_by_id returns the full account path by id
+     * the target of symlinks will be the 
+     * @param {string} account_id
+     * @returns {string} 
+    */
+    get_account_relative_path_by_id(account_id) {
+       return path.join('../', CONFIG_SUBDIRS.IDENTITIES, account_id, this.json('identity'));
+    }
+
+    /**
+     * get_account_or_user_path_by_id returns the full account/user path by id as following
+     * {config_dir}/identities/{account_id}/identity.json
+     * @param {string} account_id
+     * @returns {string} 
+    */
+    get_identity_path_by_id(account_id) {
```

Comment:
I think you need to change the argument from "account_id" to "id" and update the JSDoc description.